kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/scheduler/KafkaScheduler.java

```diff
@@ -107,43 +112,49 @@ public KafkaScheduler(KafkaSchedulerConfiguration configuration, Environment env
       new PersistentOfferRequirementProvider(frameworkState, configState, clusterState);
 
     List<Phase> phases = Arrays.asList(
-        ReconciliationPhase.create(reconciler, frameworkState),
+        ReconciliationPhase.create(reconciler),
         new KafkaUpdatePhase(
-            configState.getTargetName().toString(),
-            envConfig,
-            frameworkState,
-            offerRequirementProvider));
+                configState.getTargetName().toString(),
+                config,
+                frameworkState,
+                offerRequirementProvider));
     // If config validation had errors, expose them via the Stage.
-    Stage stage = stageErrors.isEmpty()
+    Stage installStage = stageErrors.isEmpty()
```

Comment:
`deployStage`  It's not just install.  It's also the stage ... plan that does Configuration and Software updates.